helloworld/Jenkinsfile

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/groovy
+node('redhatdistortion-maven') {
+    stage 'Checkout'
+    checkout scm
+
+    stage 'Build'
+    sh 'cd helloworld && mvn clean compile'
+
+    stage 'Run Unit Tests'
+    sh 'cd helloworld && mvn test'
+
+    stage 'Package'
+    sh 'cd helloworld && mvn package'
+
+    stage 'Archive artifact'
+    archive 'helloworld/target/*.war'
+
+    stage 'Create Image'
+    sh 'oc login https://kubernetes.default/ -u openshift-dev -p devel --insecure-skip-tls-verify=true'
```

Comment:
Hard coded password are not good - can we reuse the service account token in the pod, assuming it has correct permissions?